test/cljam/io/bam/encoder_test.clj

```diff
@@ -0,0 +1,107 @@
+(ns cljam.io.bam.encoder-test
+  "Tests for cljam.io.bam.encoder."
+  (:require [clojure.test :refer [deftest are testing]]
+            [clojure.string :as cstr]
+            [cljam.io.bam.encoder :as encoder])
+  (:import [java.nio ByteBuffer]))
+
+(deftest encode-tag-value-test
+  (testing "non-array types"
+    (are [?type ?values ?bytes]
+         (= (map unchecked-byte ?bytes)
+            (let [bb (ByteBuffer/allocate 100)]
+              (#'encoder/encode-tag-value bb ?type ?values)
+              (take (.position bb) (.array bb))))
+      \A \@ [0x40]
+      \A \A [0x41]
+      \A \z [0x7a]
+      \c 0 [0x00]
+      \c 127 [0x7f]
+      \c -128 [0x80]
+      \c -1 [0xff]
+      \C 0 [0x00]
+      \C 127 [0x7f]
+      \C 128 [0x80]
+      \C 255 [0xff]
+      \s 0 [0x00 0x00]
+      \s 32767 [0xff 0x7f]
+      \s -32768 [0x00 0x80]
+      \s -1 [0xff 0xff]
+      \S 0 [0x00 0x00]
+      \S 32767 [0xff 0x7f]
+      \S 32768 [0x00 0x80]
+      \S 65535 [0xff 0xff]
+      \i 0 [0x00 0x00 0x00 0x00]
+      \i 2147483647 [0xff 0xff 0xff 0x7f]
+      \i -2147483648 [0x00 0x00 0x00 0x80]
+      \i -1 [0xff 0xff 0xff 0xff]
+      \I 0 [0x00 0x00 0x00 0x00]
+      \I 2147483647 [0xff 0xff 0xff 0x7f]
+      \I 2147483648 [0x00 0x00 0x00 0x80]
+      \I 4294967295 [0xff 0xff 0xff 0xff]
+      \f -17.25 [0x00 0x00 0x8a 0xc1]
+      \f 0.0 [0x00 0x00 0x00 0x00]
+      \f 5.75 [0x00 0x00 0xb8 0x40]
+      \f (Float/MAX_VALUE) [0xff 0xff 0x7f 0x7f]
+      \f (Float/MIN_NORMAL) [0x00 0x00 0x80 0x00]
+      \f (Float/MIN_VALUE) [0x01 0x00 0x00 0x00]
+      \f (Float/NEGATIVE_INFINITY) [0x00 0x00 0x80 0xff]
+      \f (Float/POSITIVE_INFINITY) [0x00 0x00 0x80 0x7f]))
+
+  (testing "NULL-terminated text"
```

Comment:
I thought this part should use the new `testing` to update the description. As a result, the content of `are` in this `testing` is exactly the same as the previous one. 
The following `are` is almost the same as well.
https://github.com/chrovis/cljam/blob/c47ebe395848637db69070fb0580323b49e69fb4/test/cljam/io/bam/encoder_test.clj#L63

Do you think it is fine as it is? Or do you have any advice on how I should write it better?